pczt/src/common.rs

```diff
@@ -36,6 +39,7 @@ pub struct Global {
     /// - If omitted, the fallback locktime is assumed to be 0.
     pub(crate) fallback_lock_time: Option<u32>,
```

Comment:
Why doesn't this have `#[getset(get = "pub")]`?

Reply:
I believe that this is intended to be set only by the `Constructor` role.